src/pretix/plugins/stripe/views.py

```diff
@@ -170,20 +170,24 @@ def webhook(request, *args, **kwargs):
         rso = ReferencedStripeObject.objects.select_related('order', 'order__event').get(reference=objid)
         return func(rso.order.event, event_json, objid, rso)
     except ReferencedStripeObject.DoesNotExist:
-        if hasattr(request, 'event'):
-            return func(request.event, event_json, objid, None)
-        else:
+        if event_json['data']['object']['object'] == "charge" and event_json['data']['object']['payment_intent']:
```

Comment:
Since this is failing the automated test now, I am wondering if we should change this to 

`'payment_intent' in event_json['data']['object']`